tests/services/test_file_mover.py

```
@@ -296,10 +295,10 @@ def test_move_symlink_direct(tmp_path):
     link = tmp_path / "the_link"
     link.symlink_to(target)
     dest = tmp_path / "final_dest.txt"
-    
+
     from app.services.file_mover import _move_symlink
     success, error = _move_symlink(link, dest)
```

Ruff (0.15.7)

[warning] Unpacked variable `error` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)